Blockzilla/UIConstants.swift

```diff
@@ -182,7 +182,7 @@ struct UIConstants {
         static let navigationDoneOffset: Float = -10
         static let overlayAnimationDuration: TimeInterval = 0.25
         static let overlayButtonHeight: Int = 56
-        static let numberOfSearchSuggestions: Int = 5
+        static let numberOfSearchSuggestions: Int = 4
```

Comment:
I don't think closing this issues is as simple as changing this constant, as we don't want to cripple larger devices unnecessarily. I think we actually want to have a different max for numberOfSearchSuggestions depending on which device we're using (4 for smaller ones, 6 for larger ones). Perhaps we use two constants here and then have a check for if the user is on an iPhone 5S or any other device?

Reply:
I also think it'd be a good idea to change this to be ```maxNumberOfSearchSuggestions``` or something similar. As it stands it's a bit of a confusing name.